effectful-hpqtypes.cabal

```diff
@@ -5,14 +5,16 @@ synopsis:           Effectful bindings for hpqtypes
 
 -- A longer description of the package.
 -- description:
-homepage:
+homepage:           https://github.com/haskell-effectful/hpqtypes-effectful
 
 -- A URL where users can report bugs.
--- bug-reports:
+bug-reports:
+  https://github.com/haskell-effectful/hpqtypes-effectful/issues
+
 license:            BSD-3-Clause
 license-file:       LICENSE
-author:             Kyriakos Papachrysanthou
-maintainer:         kyriakos.papachrysanthou@scrive.com
+author:             Jakub Waszczuk
```

Comment:
I do not deserve to be mentioned as the only author :).  This was a group effort after all.  What about "Scrive AB"?

Reply:
I'd argue that you had the most part in it, but since it was a Scrive sponsored trip, let's use that.